src/lib/util/domains/domains.ts

```diff
@@ -0,0 +1,38 @@
+//- Library Imports
+import { ethers } from 'ethers';
+
+export const rootDomainId = ethers.constants.HashZero;
+
+const getSubnodeHash = (parentHash: string, labelHash: string): string => {
+	const calculatedHash = ethers.utils.keccak256(
+		ethers.utils.defaultAbiCoder.encode(
+			['bytes32', 'bytes32'],
+			[ethers.utils.arrayify(parentHash), ethers.utils.arrayify(labelHash)],
+		),
+	);
+	return calculatedHash;
+};
+
+export const getDomainId = (name?: string | null): string => {
```

Comment:
I know you didn't write this, but I'm not sure about accepting `null` as a parameter here. Using `null` here means the return will (or at least, should) be `string | null`, which means we're going to have more problems downstream.

Reply:
Updated so the prop is not optional and only takes a string.